README.rst

```diff
@@ -1,5 +1,24 @@
-BigML PHP Bindings
-=====================
+BigML PHP Bindings v. 2.0
+=========================
+
+**Notice: the BigML PHP bindings 2.0 may break existing code. In
+particular, all static methods have been removed from the BigML
+class. So, if you ever used the syntaxis, e.g.:**
+
+    BigML::create_source(...);
+
+**you will get an error. On the other hand, if you followed the syntaxis
+which was documented in this README, i.e.,**
+
+    $api->create_source(...)
+
+**you will be fine.
+
+**Additionally, notice that the old constructor which accepted all of
+its parameters as individual arguments has been deprecated in favour
+of a new one supporting the sprcification of named parameters. Use the
+new syntaxis instead, as described below. The old constructor syntaxis
```

Comment:
why? Could we use a mixed approach, leaving as individual arguments the username and api-key? probably, they are the ones that might have been used more often, so the change would affect only a small percentage.

`$api = new BigML\BigML("myusername", "my_api_key", ["storage" => "storage/data"]);`

Reply:
Well, let's do that. We'll try to give a reasonable amount of time for people to adapt.